st2actions/st2actions/config.py

```diff
@@ -73,7 +73,10 @@ def _register_action_runner_opts():
                         'Works only with Paramiko SSH runner.'),
         cfg.BoolOpt('use_ssh_config',
                     default=False,
-                    help='Use the .ssh/config file. Useful to override ports etc.')
+                    help='Use the .ssh/config file. Useful to override ports etc.'),
+        cfg.StrOpt('ssh_config_path',
+                   default='.ssh/config',
```

Comment:
Shouldn't this default to `~/.ssh/config` if we want it to use config from the user under which action runner process is running (stanley)?

Otherwise it seems it will default to a relative path based on the cwd where action runner is started.